test/index.html

```diff
@@ -10,14 +10,14 @@
 
 <body>
   <script>
-    WCT._config.environmentScripts.push('webcomponentsjs/webcomponents.js');
+    WCT._config.environmentScripts.push('webcomponentsjs/webcomponents-lite.js');
 
     WCT.loadSuites([
       'upload.html',
       'adding-files.html',
       'file-list.html'
     ].reduce(function(suites, suite) {
-      return suites.concat([suite, suite + '?dom=shadow']);
+      return suites.concat([suite, suite + '?wc-shadydom=true&dom=shadow']);
```

Comment:
This line doesn't make sense to me. Enabling shady Dom and using Shadow Dom? 

It has been recommended to me to use 

test.html?wc-shadydom=true&wc-ce=true
test.html?dom=shadow